rust/src/http/mod.rs

```diff
@@ -0,0 +1,31 @@
+use std::sync::Arc;
+
+use actix_web::HttpServer;
+use actix_web::web::Data;
+
+use crate::{AppState, database};
+use crate::cli::ServerConfig;
+use anyhow::Result;
+pub mod controllers;
+
+pub async fn start_server(args: &ServerConfig) -> Result<()> {
+
+    let session = Arc::new(database::new_session_with_keyspace(&args).await.unwrap());
```

Comment:
```suggestion
    let session = Arc::new(database::new_session_with_keyspace(args).await.unwrap());
```
reference is immediately dereferenced by the compiler